docs/Style-Guide.md

```diff
@@ -20,20 +20,94 @@ _It is still on you to ensure your code conforms to the `standard` spec, whether
 
-- Everything that can be solved by using Bulma’s classes should be solved with them
-- Theme overwrites that affect the whole application - for elements that could be found at the Bulma docs - should go to `/frontend/simple/sass` and be written in SCSS
-- Component specific styles should go to the component’s `<style>` tag, be written in SCSS, and be scoped to the component. Try to write as little of this as possible.
+- Everything that can be solved by using Bulma’s classes should be solved with them.
+- Bulma's overwrites should go to `/frontend/simple/sass/bulma_overrides` and be written in SCSS
+- Component specific styles should go to the component’s `<style>` tag, be written in SCSS and be scoped to the component. Keep this part as simple/little as possible.
+
+### How to override Bulma
+We import Bulma by modules following [its folder structure](https://github.com/jgthms/bulma/tree/master/sass). Note that the order of the modules needs to be the same to Bulma's so all CSS works correctly
+
+
+```scss
+// bulma_overrides/_index.scss
+
+@import 'utilities/index';
+@import 'base/index';
+@import 'elements/index';
+@import 'components/index';
+@import 'grid/index';
+@import 'layout/index';
+```
+
+Each folder has an `_index.scss` with all its modules imports, even those that are not being used. Those modules are commented to better understand what we use and what we don't. When a module needs to be overridden, a unique file `[moduleName].scss` is created with the needed changes and imported at `_index.scss` file. Take as example the `title` import.
+
+```scss
+// bulma_overrides/elements/_index.scss
+
+@import "../../node_modules/bulma/sass/elements/box";
+@import "../../node_modules/bulma/sass/elements/button";
+@import "../../node_modules/bulma/sass/elements/container";
+@import "../../node_modules/bulma/sass/elements/content";
+@import 'form';
+@import "../../node_modules/bulma/sass/elements/icon";
+@import "../../node_modules/bulma/sass/elements/image";
+@import 'notification';
+@import "../../node_modules/bulma/sass/elements/other";
+// @import "../../node_modules/bulma/sass/elements/progress";
+@import "../../node_modules/bulma/sass/elements/table";
+@import "../../node_modules/bulma/sass/elements/tag";
+@import 'title';
+```
+
+There are two ways of overriding Bulma:
+- **SCSS Variables** - provided by Bulma's framework.
+- **Classes** - Directly access Bulma's classes.
+
+When overriding a module, start by checking the module's documentation and see if there's a SCSS variable that does the job. Add the variables **before** the module import.
+
+```scss
+$message-radius: 1px;
+
+@import "../../node_modules/bulma/sass/components/message";
+```
+
+If there isn't a SCSS variable for that, the solution is to override directly the classes. Do that **after** the module import. When overriding a class, ask yourself if that should be modified everywhere by default or if it should be a new modifier.
```

Comment:
Well, I understand your point of view, but we Humans, are horrible predicting the future. Why do I say that:
Today it might be needed _on a particular view_ only, but since we are on a early stage, probably tomorrow we'll need it somewhere else too. When that happens we already have it available on our theme.

On the other side, we we style it locally only on the view, is more likely to be repeatable on other view/place, because a future developer might not know/remember "oh, this was used somewhere else" and then we end up with duplicated code. Even if the developer remembers it, is more likely him to "copy+paste" instead of reusing an existing class.

In a _perfect_ scenario, we would only rely on Theming classNames and reusable Vue Components without the need to style vue views files.

That's why I also prefer using it as a modifier, instead of specific style.

Reply:
As first thought, I would say nothing or only small things should change much about how to handle theming but on IMO that should be studied carefully and we need to know exactly what are our requirements/features so we build a good architecture for that. I believe that should be done on another issue.